README.md

```diff
@@ -89,6 +89,12 @@ npm run storybook
 
 Stories are deployed on [GitHub pages](https://guardian.github.io/apps-rendering)
 
-### Generating MAPI thrift files
-1. Import all types into a thrift file `v1.thrift`
-2. `node_modules/.bin/thrift-typescript --target thrift-server --outDir definitions v1.thrift`
+### Generating Apps rendering API models
+
+In the SBT repl:
+```sbtshell
+project scalaApiModels
+release
+project tsApiModels
+releaseNpm <versionNumber>
```

Comment:
could the npm version drift from the sbt package version?

Reply:
Also do you need npm credentials?

Reply:
> could the npm version drift from the sbt package version?
technically yes, although I recommend we use the same. I haven't managed to massage the scrooge plugin into releasing both scala (maven) and typescript (npm) at the same time

>Also do you need npm credentials?
Good shout I'll add that

Reply:
LGTM, can't approve as I originally opened the PR

Reply:
ha, thanks!